fred2/missionsave.cpp

```diff
@@ -768,6 +768,21 @@ int CFred_mission_save::save_asteroid_fields()
 			save_vector(Asteroid_field.inner_max_bound);
 		}
 
+		if (!Asteroid_target_ships.empty()) {
+			if (optional_string_fred("$Asteroid Targets:")) {
+				parse_comments();
+				fout(" (");
+			} else {
+				fout("\n$Asteroid Targets: (");
+			}
+
+			for (SCP_string& name : Asteroid_target_ships) {				
+				fout(" %s", name.c_str());
+			}
+
+			fout(" )");
+		}
```

Comment:
Add a `fso_comment_push(";;FSO 22.0.0;;");` to the top of this block, and change `fout("\n$Asteroid Targets: (");` to use `fout_version`.

Also put a `fso_comment_pop()` at the bottom of this block.